build.sh

```diff
@@ -71,7 +71,6 @@ function install() {
 
     cp $ROOT_DIR/build/kmesh.service /usr/lib/systemd/system/
     chmod 600 /usr/lib/systemd/system/kmesh.service
-    systemctl daemon-reload
```

Comment:
do we still need kmesh.service?

Reply:
> do we still need kmesh.service?

This is the code that was used for the rpm mode. Now it's the image mode, so it's not needed.